src/kernel-install/kernel-install.in

```diff
@@ -107,7 +107,7 @@ else
 fi
 
 # These two settings are only settable via install.conf
-layout=
+layout=auto
```

Comment:
Sorry, looking at this yesterday, I forgot about the big block at around line 250 that checks for `[ -z "$layout" ]`. The man page saying the default is `bls` reminded me of that. This should remain empty and the auto handling be moved there, because it works a few other cases as well.